### PR TITLE
VDS-545, VDS-546: Add and view products in the part of configurable product

### PR DESCRIPTION
**Pull request by ilyawzrd**

### CODE REVIEW

**Comment by ilyawzrd** [comment]
Review it after https://github.com/VirtoCommerce/vc-module-demo-features/pull/48

File: src/VirtoCommerce.DemoSolutionFeaturesModule.Web/Scripts/blades/part-product-list.js [hunk 1]
```diff
@@ -0,0 +1,211 @@
...
+        const bladeNavigationService = bladeUtils.bladeNavigationService;
+
+        blade.headIcon = 'fa-archive';
+
+        blade.currentEntity = angular.copy(blade.originalEntity);
+
+        blade.toolbarCommands = [
+            {
+                name: "platform.commands.add", icon: 'fa fa-plus',
+                executeMethod: () => $scope.addNewProduct(),
```
Comment (pushnitsa) [inline src/VirtoCommerce.DemoSolutionFeaturesModule.Web/Scripts/blades/part-product-list.js]: If method doesn't return anything or you don't want to use the returned result, you should call it with braces.
() => { $scope.addNewProduct(); }

File: src/VirtoCommerce.DemoSolutionFeaturesModule.Web/Scripts/blades/part-product-list.js [hunk 2]
```diff
@@ -0,0 +1,211 @@
...
+                        return response.items;
+                    });
+            } else {
+                $scope.pageSettings.totalItems = 0;
+                blade.currentEntities = [];
+                blade.isLoading = false;
+            }
+        };
+
+        filter.criteriaChanged = () => blade.refresh();
```
Comment (pushnitsa) [inline src/VirtoCommerce.DemoSolutionFeaturesModule.Web/Scripts/blades/part-product-list.js]: wrap method with braces

**Review by pushnitsa — CHANGES_REQUESTED** [review]

File: src/VirtoCommerce.DemoSolutionFeaturesModule.Web/Scripts/blades/part-product-list.js (hunk 1 above)
Comment (ilyawzrd) [reply src/VirtoCommerce.DemoSolutionFeaturesModule.Web/Scripts/blades/part-product-list.js]: done

File: src/VirtoCommerce.DemoSolutionFeaturesModule.Web/Scripts/blades/part-product-list.js (hunk 2 above)
Comment (ilyawzrd) [reply src/VirtoCommerce.DemoSolutionFeaturesModule.Web/Scripts/blades/part-product-list.js]: done

**Review by pushnitsa — DISMISSED** [review]

**Review by trueboroda — DISMISSED** [review]

**Review by pushnitsa — APPROVED** [review]

**Pull request merged**